### PR TITLE
Update on-boarding guides

### PR DESCRIPTION
**Pull request by amyblais**
- Added steps to on-boarding guides.
- Added a few new missing hires to list of R&D folks.

### CODE REVIEW

File: site/content/internal/onboarding/manager-guide.md [hunk 1]
```diff
@@ -32,4 +32,10 @@ weight: 30
 - Add new staff to the group
 
 ### Core Committer Mug
-- Message @lindy and ask her to send the new hire their core committer mug
+- Message @lindy.isherwood and ask her to send the new hire their core committer mug
```
Comment (enahum) [inline site/content/internal/onboarding/manager-guide.md]: this is now done by @hannaparks apparently

Comment (jwilander) [reply site/content/internal/onboarding/manager-guide.md]:
```suggestion
- Message @hanna.park and ask her to send the new hire their core committer mug
```

File: site/content/internal/onboarding/manager-guide.md [hunk 2]
```diff
@@ -32,4 +32,10 @@ weight: 30
...
+
+### Release Dates Google Calendar
+- Message @lindy.isherwood and ask her to add the new hire to the Mattermost Release Dates google calendar
+
+### Jira Access
+- Message @amy.blais and ask her to add the new hire to ``jira-developers`` and ``internal`` groups in Jira
```
Comment (lieut-data) [inline site/content/internal/onboarding/manager-guide.md]: I'm assuming we'll automate this at some point?

Comment (amyblais) [reply site/content/internal/onboarding/manager-guide.md]: That's a good point, I'll add a note for myself as next steps.

**Comment by amyblais** [comment]
Elias, I can add a note for myself to consider creating a channel for this as a next step. For now I want to make sure these steps are included in the checklist so they're not forgotten.

**Pull request merged**